samples/samples-js/GetProductNamesView/index.js

```diff
@@ -0,0 +1,12 @@
+module.exports = async function (context, req, products) {
+    context.log('JavaScript HTTP trigger function processed a request.');
+    if (products.length === 0) {
+        context.log("Product(s) not found");
+    } else {
+        context.log("Found Product(s), Count=" + products.length + " -> " + JSON.stringify(products));
+    }
+    return {
+        status: 201,
```

Comment:
201 is created which isn't correct, this should be 200 (go through and verify all functions are returning the correct code)